integration_test/gce/gce_testing.go

```diff
@@ -1792,8 +1834,8 @@ type VMOptions struct {
 // If VM creation fails, it will abort the test.
 // At the end of the test, the VM will be cleaned up.
 func SetupVM(ctx context.Context, t *testing.T, logger *log.Logger, options VMOptions) *VM {
+	options.ImageFamily = options.Platform // Temporary until 'Platform' is fully replaced.
```

Comment:
This is redundant with the same thing being done inside CreateInstance, just delete it from here.